### PR TITLE
GCLP-773 multiple check level percentage discount

### PR DESCRIPTION
**Pull request by acai-toast**

### CODE REVIEW

File: accounts.js [hunk 1]
```diff
@@ -244,6 +244,11 @@ function inquireOrRedeem(identifier, check, redemptions, transactionType) {
     }
   }
 
+  var preDiscounntAmount = check.amount;
```
Comment (ltavis-toast) [inline accounts.js]: I think there is business logic guidelines that we need to communicate to partners. Can you write up how we recommend that partners process multiple check level discounts.  And share that information via a different ticket or this ticket with Sharmita.

Comment (acai-toast) [reply accounts.js]: Sure, but the partner does not need to follow the same rule

Comment (ltavis-toast) [reply accounts.js]: If they don't follow similar rule, will the original bug show up about a discount being removed/adjusted?

Comment (acai-toast) [reply accounts.js]: No, if they do the math correctly, they won't have this bug. We do the math wrong on the reference implementation and the POS handles it as expected

Comment (ltavis-toast) [reply accounts.js]: ok

**Pull request merged**